tests/sles4sap/hana_cluster.pm

```diff
@@ -76,7 +76,7 @@ sub run {
         barrier_wait "HANA_INIT_CONF_$cluster_name";
 
         assert_script_run "su - $sapadm -c 'sapcontrol -nr $instance_id -function StopSystem HDB'";
-        sleep 10;
+        sleep 60;
```

Comment:
That's help but maybe only for this case (I tried also on my lab, but with a smaller value, which didn't helped), I think it's better to clearly check for a stop status.

`hdbnsutil -sr_state` shows this when process is up:

    [ .... ]
    System Replication State
    ~~~~~~~~~~~~~~~~~~~~~~~~
    online: true
    [ .... ]

And when it is off we have this: `online: false`, so maybe better to check for this value instead of an arbitrary sleep, WDYT?

Reply:
Hope you like it now :)